webui/react/package.json

```diff
@@ -110,24 +110,24 @@
     "@types/sprintf-js": "^1.1.2",
     "@types/throttle-debounce": "^2.1.0",
     "@types/webpack": "^5.28.0",
-    "@typescript-eslint/eslint-plugin": "^5.36.2",
-    "@typescript-eslint/parser": "^5.36.2",
+    "@typescript-eslint/eslint-plugin": "^5.40.0",
+    "@typescript-eslint/parser": "^5.40.0",
     "antd-dayjs-webpack-plugin": "^1.0.1",
     "antd-theme-generator": "^1.2.11",
     "antd-theme-webpack-plugin": "^1.3.9",
     "babel-plugin-import": "^1.13.5",
     "craco-less": "^1.20.0",
     "craco-sass-resources-loader": "^1.1.0",
-    "eslint": "^7.25.0",
+    "eslint": "^8.25.0",
```

Comment:
I think 8.25 has breaking change (not sure it affects us tho).
No need to change eslint config?